node/src/components/block_executor.rs

```diff
@@ -159,6 +169,8 @@ impl BlockExecutor {
         // The state hash of the last execute-commit cycle is used as the block's post state
         // hash.
         let next_height = state.finalized_block.height() + 1;
+        // Update the metric.
+        self.metrics.chain_height.set(next_height as i64);
```

Comment:
Shouldn't we use _this_ block's height instead of the next?

Reply:
Oh good catch, my mistake, I shall rectify